.gitignore

```diff
@@ -1,10 +1,10 @@
 # build and test outputs
-bin/
-_output/
-_artifacts/
+/bin/
+/_output/
+/_artifacts/
 
 # used for the code generators only
-vendor/
+/vendor/
```

Comment:
> If there is a separator at the beginning or middle (or both) of the pattern, then the pattern is relative to the directory level of the particular .gitignore file itself. Otherwise the pattern may also match at any level below the .gitignore level.

https://git-scm.com/docs/gitignore

Reply:
(we were matching `images/base/files/usr/local/bin/`)

Reply:
:man_shrugging: